dumb-jump.el

```diff
@@ -2111,6 +2108,13 @@ to keep looking for another root."
         (thing-at-point 'symbol)
       (thing-at-point 'symbol t))))
 
+(defun dumb-jump--get-symbol-start ()
+  "Get bounds of symbol at point"
+  (- (if (region-active-p)
+         (region-beginning)
```

Comment:
I'm not sure if using the beginning of the region is the expected behaviour, especially for xref.

Reply:
`dumb-jump-get-point-symbol` returns the region (if it is active) for both normal and xref mode. So we want to model the same behavior.

Reply:
You're right, I forgot that `dumb-jump-get-point-symbol` used `region-active-p`.